registries.conf

```diff
@@ -25,33 +16,6 @@
 # If it is necessary to use one of these registries, it should be added at the
 # end of the list.
 #
-# It is recommended to use fully-qualified images for pulling as the
-# destination registry is unambiguous. Pulling by digest
-# (i.e., quay.io/repository/name@digest) further eliminates the ambiguity of
-# tags.
```

Comment:
Shouldn’t this paragraph stay?

Reply:
Although after expanding the verbiage that GitHub auto hid, I think it's OK to remove this as it's a bit redundant.

Reply:
I squeezed it in the initial note.